source/image-handler/package.json

```diff
@@ -15,7 +15,8 @@
     "color": "4.2.3",
     "color-name": "1.1.4",
     "sharp": "^0.31.3",
-    "aws-sdk": "^2.1356.0"
+    "aws-sdk": "^2.1356.0",
+    "jimp": "^0.22.8"
```

Comment:
This package brings a lot of dependencies, for BMP it uses https://github.com/shaozilee/bmp-js, please refer to https://github.com/jimp-dev/jimp/blob/245f1cf60e0167d4d733ef90e638428d065856b2/packages/type-bmp/package.json#L25 I think you could use `bmp-js` instead.